README.md

```diff
@@ -1 +1,25 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## About me
+**Hi --- I'm Neeraj!** I just finished my first year at UChicago, and am an Economics and Political Science double major. I honestly haven't done much work in my majors thus far, and have instead opted to do mostly core throughout my first year. I'm enrolled in the summer quarter as part of the Institute in the Social Research Methods. If I had to guess, that's why most of the undergraduates are also enrolled in this course. After the Summer quarter classes end, I'll be RAing for Professor Mark Bradley, Professor in the History Department. I'm going to be applying the computational social science skills I gain in this class to identify meaningful relationships between the policy actions and rhetoric surrounding the ideas surrounding the 'Third World' and 'Global South.'
```

Comment:
Best practice is to leave a blank line after a header before the content. It will prevent rendering issues with more complex R Markdown documents